src/webgpu/api/validation/createBindGroupLayout.spec.ts

```diff
@@ -54,10 +55,6 @@ g.test('duplicate_bindings')
     }, !_valid);
   });
 
-// MAINTENANCE_TODO: Move this into kLimits with the proper name after the spec PR lands.
-// https://github.com/gpuweb/gpuweb/pull/3318
-const kMaxBindingsPerBindGroup = 640;
```

Comment:
For reviewers: That PR has been already merged so I removed this line. Instead kLimitInfo is imported.